trafilatura/metaxpaths.py

```diff
@@ -5,48 +5,65 @@
 # code available from https://github.com/adbar/trafilatura/
 # under GNU GPLv3+ license
 
+attr_types = {
+    'class': 'translate(@class,"ABCDEFGHIJKLMNOPURSTUWXYZ","abcdefghijklmnopurstuwxyz")',
+    'id': 'translate(@id,"ABCDEFGHIJKLMNOPURSTUWXYZ","abcdefghijklmnopurstuwxyz")',
+    'itemprop': 'translate(@itemprop,"ABCDEFGHIJKLMNOPURSTUWXYZ","abcdefghijklmnopurstuwxyz")',
+    'rel': 'translate(@rel,"ABCDEFGHIJKLMNOPURSTUWXYZ","abcdefghijklmnopurstuwxyz")',
+}
 
 author_xpaths = [
-    '//*[(self::a or self::address or self::link or self::p or self::span)][@rel="author" or @id="author" or @class="author" or rel="me"]|//author',
-    '//*[(self::a or self::span)][contains(@class, "authors") or contains(@class, "author") or contains(@class, "posted-by") or contains(@itemprop, "author")]',
-    '//*[(self::a or self::div or self::p or self::span)][contains(@class, "byline")]',
-    '//*[contains(@class, "author") or contains(@class, "screenname")]',
+    '//*[(self::a or self::address or self::link or self::p or self::span)][' + attr_types['rel'] + '="author" or ' +
+    attr_types['id'] + '="author" or ' + attr_types['class'] + '="author" or ' + attr_types['rel'] + '="me"]|//author',
+    '//*[(self::a or self::span)][contains(' + attr_types['class'] + ', "authors") or (contains(' + attr_types[
+        'class'] + ', "author") and not(contains(' + attr_types['class'] + ', "prefix"))) or contains(' + attr_types[
+        'class'] + ', "posted-by") or contains(' + attr_types['itemprop'] + ', "author")]',
+    '//*[(self::a or self::div or self::p or self::span)][contains(' + attr_types[
+        'class'] + ', "byline") or contains(' + attr_types['class'] + ', "journalist")]',
+    '//*[contains(' + attr_types['class'] + ', "author") or contains(' + attr_types['class'] + ', "screenname")]',
 ]
 
-
 categories_xpaths = [
-    """//div[starts-with(@class, 'post-info') or starts-with(@class, 'postinfo') or
-    starts-with(@class, 'post-meta') or starts-with(@class, 'postmeta') or
-    starts-with(@class, 'meta') or starts-with(@class, 'entry-meta') or starts-with(@class, 'entry-info') or
-    starts-with(@class, 'entry-utility') or starts-with(@id, 'postpath')]//a""",
-    "//p[starts-with(@class, 'postmeta') or starts-with(@class, 'entry-categories') or @class='postinfo' or @id='filedunder']//a",
-    "//footer[starts-with(@class, 'entry-meta') or starts-with(@class, 'entry-footer')]//a",
-    '//*[(self::li or self::span)][@class="post-category" or @class="postcategory" or @class="entry-category"]//a',
-    '//header[@class="entry-header"]//a',
-    '//div[@class="row" or @class="tags"]//a',
+    '//div[starts-with(' + attr_types['class'] + ', "post-info") or starts-with(' + attr_types[
```

Comment:
This PR is about authors, I'd rather leave the three other XPath changes to another PR.